src/ssl_bench/methods/FixMatch.py

```
@@ -0,0 +1,54 @@
+from ssl_bench.methods.MethodWrapper import MethodWrapper
+from torch.nn import functional as func
+import torchvision.transforms.v2 as transforms
+import torch
+
+
+class FixMatch(MethodWrapper):
+    def forward(self, l_data, l_labels, u_data=None):
+        self.backbone.train()
+
+        if u_data is None:
+            print("Warning: u_data is None")
```

I think this should just raise an Exception with a more detailed message (i.e. "FixMatch requires unlabeled data") to stop things from running. As far as I know we never want to use the mix with zeros method used as a fallback here.